diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter1/EnumValueCaseTest.kt

```diff
@@ -2,12 +2,14 @@ package org.cqfn.diktat.ruleset.chapter1
 
 import org.cqfn.diktat.ruleset.rules.IdentifierNaming
 import org.cqfn.diktat.util.FixTestBase
+import org.junit.jupiter.api.Tag
 import org.junit.jupiter.api.Test
 
 class EnumValueCaseTest : FixTestBase ("test/paragraph1/naming", IdentifierNaming()){
 
     @Test
-    fun `incorrect enum value (fix)`() {
+    @Tag("ENUM_VALUE")
```

Comment:
I think the idea was to have a single place with all warning names as string constants. This way we will have to duplicate strings only once and then reuse them, instead of copy-pasting everuwhere.